src/main/kotlin/net/maizegenetics/phgv2/api/ExportHaplotypeGraph.kt

```diff
@@ -0,0 +1,118 @@
+package net.maizegenetics.phgv2.api
+
+import biokotlin.seq.NucSeqRecord
+import biokotlin.seqIO.NucSeqIO
+import htsjdk.variant.variantcontext.Allele
+import htsjdk.variant.variantcontext.GenotypeBuilder
+import htsjdk.variant.variantcontext.VariantContext
+import htsjdk.variant.variantcontext.VariantContextBuilder
+import htsjdk.variant.variantcontext.writer.Options
+import htsjdk.variant.variantcontext.writer.VariantContextWriterBuilder
+import htsjdk.variant.vcf.VCFAltHeaderLine
+import htsjdk.variant.vcf.VCFHeader
+import htsjdk.variant.vcf.VCFHeaderLine
+import net.maizegenetics.phgv2.utils.altHeaderMetadataToVCFHeaderLine
+import net.maizegenetics.phgv2.utils.createGenericHeaderLineSet
+import org.apache.logging.log4j.LogManager
+import java.io.File
+
```

Comment:
Add header comment to file (because all files should have them).

Reply:
Ok I added that